src/core/constraint_template.jl

```diff
@@ -253,6 +253,29 @@ function constraint_mc_power_balance_shed(pm::AbstractUnbalancedPowerModel, i::I
 end
 
 
+"KCL with capacitor control variables."
```

Comment:
should correct to follow new conventions for docstring format that include the function signature